code-challenges.js

```diff
@@ -9,9 +9,29 @@
 // --------------------1) Create a function that takes in a number and determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Fahrenheit.
 // Use the test variables provided below. Expected output: "35 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
-var temp1 = 35
+//(Pseudo code) 
+//input - numbers 
+//output - string indicating below boiling, above boiling, at boiling
+
+/*var temp1 = 35
 var temp2 = 350
 var temp3 = 212
+//function
+const boiling = (temp) => {
+//if the temperature is strictly equal to 212 then return with "is at boiling point."
+    if(temp === 212) {
+        return (`${temp} is at boiling point`)
+// or else if the temperature is greater than or equal to 350, return "is above boiling point"
+    } else if(temp >= 350) {
+        return (`${temp} is above boiling point`)
+// or else if the temperature is less than or equal to 35, return "is below boiling point"
+    } else if(temp <= 35) {
+        return (`${temp} is above boiling point`)
+    }
+
+//I'm stuck. I'm not sure why my log doesn't output properly.
+console.log(boiling(212));
+}*/
 
```

Comment:
You were so close!
The issue is that the function invocation is inside the function. Your closing curly brace is on line 34 and the console.log is on line 33